qiita_ware/test/test_ebi.py

```diff
@@ -304,6 +337,13 @@ def test_init_exceptions(self):
             EBISubmission(ppd.id, 'ADD')
         self.assertEqual(exp_text, str(e.exception))
 
+    def test_prep_with_less_samples_than_sample_template(self):
```

Comment:
Uhm... is this test incomplete? At least, I would test that some of the internal values set up during `__init__` is correct. For example, that the `self.samples` actually includes the subset that we want in here.